serving/src/main/java/feast/serving/util/Metrics.java

```diff
@@ -40,20 +40,12 @@
 
   public static final Counter missingKeyCount =
```

Comment:
Maybe just rename this to match the metric name?

Reply:
Renamed.